scipy/spatial/_spherical_voronoi.py

```diff
@@ -216,16 +141,26 @@ class SphericalVoronoi:
 
     """
 
-    def __init__(self, points, radius=None, center=None, threshold=1e-06):
+    def __init__(self, points, radius=1, center=(0, 0, 0), threshold=1e-06):
+        if radius is None:
+            warnings.warn('`radius` is `None`. '
+                          'This will raise an error in a future version. '
+                          'Please provide a floating point number '
+                          '(i.e. `radius=1`).',
+                          DeprecationWarning)
+        if center is None:
+            warnings.warn('`center` is `None`. '
+                          'This will raise an error in a future version. '
+                          'Please provide a coordinate '
+                          '(i.e. `center=(0, 0, 0)`)',
+                          DeprecationWarning)
+        if radius is None:
```

Comment:
any reason we need to repeat the conditional checks for `None` here? Can't we just issue the warning *and* assign the values?

Reply:
Yep.  That was silly.  It is fixed now.